Dockerfile.dapper

```diff
@@ -10,21 +10,16 @@ ENV DOCKER_BUILDX_URL_arm=https://github.com/docker/buildx/releases/download/v0.
     DOCKER_BUILDX_URL=DOCKER_BUILDX_URL_${ARCH}
 RUN wget -O - ${!DOCKER_BUILDX_URL} > /.docker/cli-plugins/docker-buildx && chmod +x /.docker/cli-plugins/docker-buildx
 
-FROM ubuntu:18.04
+FROM rancher/hardened-build-base:v1.16.4b7
```

Comment:
```suggestion
FROM rancher/hardened-build-base:v1.16.7b7
```

Reply:
oh yeah I always forget it about it, fixing now